### PR TITLE
fix: add styles to Delivery View header

### LINKED ISSUE #125: Map Page in Desktop: The header is decentralized
**Describe the bug**
After the last Pull Request, the header of the map looks decentralized. Look good on screens of < 600px

**To Reproduce**
1. Go to https://deploy-preview-122--azordev-dasher-user.netlify.app/delivery/test

**Expected behavior**
Back Arrow at the left, text at the center and chat button at the right

**Screenshots**
imagen

### Checklist

- [ ] Navbar look good on screens > to 600px
- [ ] Navbar look good on screens > to 900px(desktop)

### PR DESCRIPTION
**Pull request by denisosuna**
### Description

The header elements  in the chat and delivery layout were centered
It Was added state to change the current status of the modal on the delivery page 
Added router back component in confirm Page

fixes #125 #126 #127 

#### Screenshots

image
image
image

#### Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

### How to test it

If applicable, describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

### Checklist:

- [ ] Doesn't break the current code.
- [ ] Passes linters and test, also is building.
- [ ] Doesn't have spelling or grammatical problems.
- [ ] Doesn't have unnecessary comments or debugging code.
- [ ] The branch is updated with the last dev branch changes.

### CODE REVIEW

**Review by heliomar-pena — APPROVED** [review]
I see all good. Good job!

File: src/pages/Confirm/Confirm.layout.jsx [hunk 1]
```diff
@@ -1,13 +1,12 @@
 import PropTypes from 'prop-types'
 import { Text } from '../../components'
 import note from '../../assets/note.png'
-import arrow from '../../assets/arrow-left.svg'
 import { Container, Header, MainSection } from '../../layouts/Splitted.styled'
-
+import BackButton from '../../components/BackButton'
```
Comment (Israel-Laguan) [inline src/pages/Confirm/Confirm.layout.jsx]: Move this to line 2, to be along the other components

File: src/pages/Delivery/Delivery.layout.jsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import PropTypes from 'prop-types'
 import { Text } from '../../components'
-import { useHistory } from 'react-router-dom'
+import {  useHistory } from 'react-router-dom'
```
Comment (Israel-Laguan) [inline src/pages/Delivery/Delivery.layout.jsx]: extra space

File: src/pages/Delivery/Delivery.styled.jsx [hunk 3]
```diff
@@ -33,15 +34,17 @@ const HeaderText = styled.div`
   margin: auto 0;
   height: auto;
 
-  @media (min-width: 960px) {
+
```
Comment (Israel-Laguan) [inline src/pages/Delivery/Delivery.styled.jsx]: extra space

File: src/pages/Delivery/Delivery.styled.jsx [hunk 4]
```diff
@@ -33,15 +34,17 @@ const HeaderText = styled.div`
...
+  @media (min-width: 900px) {
     font-size: 2em;
   }
 
   @media (min-width: 3180px) {
     font-size: 6em;
   }
 
-  @media (max-width: 960px) {
+  @media (max-width: 889px) {
```
Comment (Israel-Laguan) [inline src/pages/Delivery/Delivery.styled.jsx]: why is this needed?

File: src/components/Rating/Rating.jsx [hunk 5]
```diff
@@ -1,7 +1,7 @@
 import { StarBtn, SelectedStarBtn } from './Rating.styled'
 import { useState } from 'react'
 
-const Rating = () => {
+const Rating = (z) => {
```
Comment (Israel-Laguan) [inline src/components/Rating/Rating.jsx]:
```suggestion
const Rating = () => {
```

**Review by Israel-Laguan — APPROVED** [review]
